client/Lykke.Service.OperationsCache.Client/OperationsCacheClient.cs

```diff
@@ -11,15 +11,21 @@ namespace Lykke.Service.OperationsCache.Client
 {
     public class OperationsCacheClient : IOperationsCacheClient, IDisposable
     {
-        private readonly ILog _log;
+        private readonly ILog _log; // Unused. Left for compatibility.
```

Comment:
Compatibility with what? Since this is private field, it should be removed. Otherwise, someone will use it in futher updates and this will lead to the NullReferenceException, if new ctor will be used